crates/dbs-virtio-devices/src/fs/device.rs

```diff
@@ -62,12 +61,6 @@ pub(crate) const PASSTHROUGHFS: &str = "passthroughfs";
 pub(crate) const BLOBFS: &str = "blobfs";
 pub(crate) const RAFS: &str = "rafs";
 
-#[derive(Clone, Deserialize)]
-struct BlobCacheConfig {
```

Comment:
Is this useless?

Reply:
Yes, in the new Nydus-api, they introduce `ConfigV2`, it could be created through `RafsConfig` str. `blob_config` is directly generated in the creation of `ConfigV2`, so there is no need for dbs-virtio-devices to define and create the BlobCacheConfig for Nydus.